daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/NodeInfo.scala

```diff
@@ -85,4 +85,16 @@ object NodeInfo {
       keyMaintainers
   }
 
+  trait Rollback extends NodeInfo {
+
+    final def requiredAuthorizers: Set[Party] =
+      // TODO https://github.com/digital-asset/daml/issues/8020
+      sys.error("rollback nodes are not supported")
+
+    final def informeesOfNode: Set[Party] =
+      // TODO https://github.com/digital-asset/daml/issues/8020
+      sys.error("rollback nodes are not supported")
+
+  }
```

Comment:
I am not sure it is worthwhile to add that now.

Reply:
These are also needed to fulfil the `GenNode` interface.